src/App.js

```diff
@@ -7,74 +7,116 @@ import ListOfMembers from "./components/ListOfMembers";
 import ListOfTrainers from "./components/ListOfTrainers";
 import CreateMembers from "./Pages/CreateMembers";
 import ViewMember from "./Pages/ViewMemeber"
-import EditMember from "./Pages/EditMember";
+import EditMember from "./Pages/EditMemberForm";
 import Header from "./components/Header"
+import EditClassForm from "./Pages/EditClassForm";
+
 
 export default function App() {
   const [trainers, setTrainers] = useState([])
   const [classes, setClasses] = useState([])
   const [members, setMembers] = useState([])
-  const [memberToView ,setMemberToView] = useState([])
+  const [detailsToEdit, setDetailsToEdit] = useState({})
+  const [contactEdit, setContactEdit] = useState(false)
+  const [memberToView, setMemberToView] = useState([])
+  const [profile,setProfile ] = useState([]) 
+  const [address,setAddress ] = useState([])
+
 
   const API_URL = process.env.REACT_APP_API_URL;
 
   console.log("Inside App State: ", {
     trainers,
     classes,
-    members
+    members,
+    detailsToEdit,
+    contactEdit
   })
 
-  useEffect (() => {
+  useEffect(() => {
```

Comment:
Using multiple useEffect hooks in a component is fine, but it's a little cleaner to stick to one where possible. A good solution here is to separate all your fetch calls into their own functions, then call each of the functions in the one useEffect. E.g.

```
function fetchClasses() {
    fetch(`${API_URL}/classes`)
      .then((res) => res.json())
      .then((classData) => {
        setClasses(classData);
      });
}

function fetchMembers() {}
function fetchTrainers() {}

useEffect(() => {
    fetchClasses();
    fetchMembers();
    fetchTrainers();
}, [])
```

Reply:
I will effect this change and use just one useEffect